elliptic-curve/src/arithmetic.rs

```diff
@@ -84,3 +84,14 @@ pub trait PrimeCurveArithmetic:
     /// Prime order elliptic curve group.
     type CurveGroup: group::prime::PrimeCurve<Affine = <Self as CurveArithmetic>::AffinePoint>;
 }
+
+/// Normalize point(s) in projective representation by converting them to their affine ones.
+pub trait Normalize<Points>: group::Curve {
```

Comment:
If you want the method to be `batch_normalize`, then the trait name should probably match if it's the only method:

```suggestion
pub trait BatchNormalize<Points>: group::Curve {
```